packages/contracts/src/framework/plugin/repo/PluginRepoFactory.sol

```diff
@@ -101,6 +102,11 @@ contract PluginRepoFactory {
             address(this),
             maintainerPermissionID
         );
+        items[5] = PermissionLib.SingleTargetPermission(
+            PermissionLib.Operation.Revoke,
+            address(this),
+            upgradePermissionID
+        );
```

Comment:
For me, it is still a bit off design-wise that `PluginRepoFactory` gets permissions that it doesn't need. However, because we don't want to have two `initialize` functions, it is what it is.

Reply:
Are we really using the upgrade permission in the pluginRepoFactory at all? 
I don't see the reason to drop a permission for something that might not be needed in the first place